tsconfig.node-base.json

```diff
@@ -61,6 +61,15 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
 
     /* Advanced Options */
-    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
+    "forceConsistentCasingInFileNames": true, /* Disallow inconsistently-cased references to the same file. */
+    "lib": [
+      "DOM",
+      "DOM.Iterable",
+      "ES6",
+      "ES2017",
+      "ES2019",
+      "ES2019.array",
+      "ScriptHost"
```

Comment:
We still support node v10 right now. And I'm not sure node v12 supports es2019 features.

Reply:
flatMap starts at node v11

I can polyfill at the top of the file, it's only a few lines